Tests/Core/KSPManager.cs

```diff
@@ -109,6 +109,37 @@ public void AddInstance_ManagerHasInstance()
             }
         }
 
+        [Test]
+        public void CloneInstance_BadInstance_ThrowsNotKSPDirKraken()
```

Comment:
This test seems to be failing.

https://travis-ci.org/KSP-CKAN/CKAN/jobs/473907148#L2592

Reply:
Yep, found the mistake already and is fixed in the next commit.